### ISSUE TITLE
dist_an.append(torch.tensor([dist[i][mask[i] == 0].min()

### ISSUE THREAD

**Issue by zhanghanqi** [yulu0724/EmbeddingDistillation#4]
dist_an.append(torch.tensor([dist[i][mask[i] == 0].min().unsqueeze(0)]))
invalid argument 1: cannot perform reduction function min on tensor with no elements because the operation does not have an identity 
I use the code to instead of it.
            if len([dist[i][mask[i] == 0]]) == 0:
                dist_an.append([1e+12])
            else:
                dist_an.append(torch.tensor([dist[i][mask[i] == 0].min().unsqueeze(0)]))
But this code still has the same code.
